packages/plugin-core/src/workspace/tutorialInitializer.ts

```diff
@@ -61,36 +54,28 @@ export class TutorialInitializer
 
     const vpath = vault2Path({ vault: opts.vaults[0], wsRoot: opts.wsRoot });
 
-    fs.copySync(path.join(dendronWSTemplate.fsPath, "tutorial"), vpath);
-
-    const ABUserGroup = MEETING_NOTE_TUTORIAL_TEST.getUserGroup(
+    const ABUserGroup = AB_2022_05_REFACTOR_MENTION.getUserGroup(
```

Comment:
instead of AB testing specifically for refactor mention, maybe this should just be a long running ab test for tutorial changes? `eg. AB_TUTORIAL_TEST. the groups here can change over time with different values that represent what we're testing

Reply:
Nice idea. That will make updating the tutorial easier each time since we don't have to ever touch `tutorialInitializer`. We just modify the test group.

@jonathanyeung thoughts?

Reply:
I think in this situation it _should_ work. 

For other situations, I don't think this is good practice though, as modifying the number of groups or their weights will cause people to change groups over time, which will muddy any findings.  Given that users most likely only go through the tutorial once, it shouldn't be much of an issue here.

Reply:
I think one way we can tackle this (if we want to take care of the case where user goes through tutorial multiple times during their use of Dendron) is to track what group the user was initially assigned to in metadata store.

```
lastTutorialTestGroup = metadata.tutorialTestGroup
if AB_TUTORIAL_TEST[lastTutorialTestGroup] exists:
  show tutorial for lastTutorialTestGroup regardless of test group change
else
  setMeta(newlyAssignedTestGroup) # track it for the next round
  handleCurrentTestGroup
```

This will at least preserve what we show the user between AB tests if we keep some groups around from last time as baseline.

Reply:
but like @jonathanyeung said, user will most likely only go through the tutorial once, so I'll just keep this noted and not implemented for the time being.

Reply:
@hikchoi @kevinslin On second thought, it's better to never change the groups.  Which AB group a user is in is added to User Props, so if a user changes groups, we won't be able to see their historical data properly because their user props have changed (at least not easily in Amplitude).

We can defer making a decision until the next time around, so I have no problems to go ahead and merge as-is.